help/policies/_posts/2024-02-12-mirrors.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "ADS Mirror Links Update"
```

Comment:
Again, make this sound more policy-like; something like: "ADS Mirror Site Policy"